kernels/wmma_extension.h

```diff
@@ -57,6 +69,37 @@ inline __device__ void bmma_sync(
   }
 }
 
+#if defined(__GFX9__) || defined(__GFX12__) || __CUDA_ARCH__ >= 890
```

Comment:
```suggestion
#if __CUDA_ARCH__ >= 890
```

AMD has a different syntax for intrinsics and different matrix layouts.